docs/_includes/generated/all_taskmanager_section.html

```diff
@@ -18,7 +18,7 @@
             <td><h5>task.cancellation.timeout</h5></td>
             <td style="word-wrap: break-word;">180000</td>
             <td>Long</td>
-            <td>Timeout in milliseconds after which a task cancellation times out and leads to a fatal TaskManager error. A value of 0 deactivates the watch dog.</td>
+            <td>Timeout in milliseconds after which a task cancellation times out and leads to a fatal TaskManager error. A value of 0 deactivates the watch dog. Notice that a task cancellation is different from a task failure. So task cancellation timeout does not apply to task closing/clean-up caused by a task failure.</td>
```

Comment:
> Notice that a task cancellation is different from a task failure. So task cancellation timeout does not apply to task closing/clean-up caused by a task failure

->

> Notice that a task cancellation is different from both a task failure and a clean shutdown. So task cancellation timeout applies only if you manually cancel the job and does not apply to task closing/clean-up caused by a task failure or clean shutdown.

? Because as I understand, this notice applies as well for the clean shut down?

Reply:
1. That's true in the sense that `cleanUpInvoke` is called both in the case of a task failure and a clean shutdown. But I thought `task-cancelation-timeout` naturally can not be considered as applied to a clean shutdown? The main confusion in general (and in the ticket FLINK-18983) is that people thought "task-cancelation-timeout" can be applied for "failed tasks" as well.

2. Task cancelation does not always happen "manually", it can happen when a task fails and caused the rest of the tasks canceled by JM. So I would say "task cancelation" instead of saying "manually cancel the job"

Reply:
> > Notice that a task cancellation is different from a task failure. So task cancellation timeout does not apply to task closing/clean-up caused by a task failure
> 
> ->
> 
> > Notice that a task cancellation is different from both a task failure and a clean shutdown. So task cancellation timeout applies only if you manually cancel the job and does not apply to task closing/clean-up caused by a task failure or clean shutdown.
> 
> ? Because as I understand, this notice applies as well for the clean shut down?

1. That's true in the sense that cleanUpInvoke is called both in the case of a task failure and a clean shutdown. But I thought task-cancelation-timeout naturally can not be considered as applied to a clean shutdown? The main confusion in general (and in the ticket FLINK-18983) is that people thought "task-cancelation-timeout" can be applied for "failed tasks" as well.

2. Task cancelation does not always happen "manually", it can happen when a task fails and caused the rest of the tasks canceled by JM. So I would say "task cancelation" instead of saying "manually cancel the job"

Reply:
1. I would suggest to add clean shutdown mention, because up till the moment I started reading your PR, I was sure that the FLINK-18983 is all about clean shutdown 😓 So the mis conception can go in both directions .
2. 👍